tracks/python/exercises/raindrops/mentoring.md

```diff
@@ -1,37 +1,20 @@
 ### Problem and Challenges
 
-The problem asks us to take an integer and return a string.
+The problem asks us to take an **int** and return a **str**.
 We need to map potential factors 3, 5, and 7 to Pling, Plang, and Plong.
 If the integer is not divisible by any of the factors, we must
 return a string representing the number in base 10.
 
 ### Reasonable Solutions
 
-```python
-def raindrops(val: int) -> str:
-
-    sound_map = [
-        (3, 'Pling'),
-        (5, 'Plang'),
-        (7, 'Plong'),
-        ]
-
-    lst = [sound for factor, sound in sound_map if not (val % factor)]
+This exercise is an early core exercise and is intended to introduce
+simple concepts such as basic conditionals, modulo division, and string 
+concatenation.
 
-    if not lst:
-        result = str(val)
-    else:
-        result = ''.join(lst)
-
-    return result
-```
-
-### Common Suggestions
-
-#### Linear testing
+Therefore the following is considered both reasonable and acceptable:
 
 ```python
-def raindrops(number): 
+def convert(number: int) -> str:
```

Comment:
Is is worthwhile to include some comment on how this is not valid in Python 2? I know we are encouraging Python 3, but since 2 is technically still supported, some students will be using it and may run into issues if a mentor suggests this syntax.

Reply:
I'll address this with a Talking Point, but my inclination is to have all the examples in Python 3 and just let Python 2 be handled by the mentor as they discover the mentee is using it ... a SyntaxError there might very well be the first indication that's the situation.